runtime/src/accounts_db.rs

```diff
@@ -2097,7 +2097,7 @@ impl AccountsDb {
         );
     }
 
-    fn do_shrink_slot_stores<'a, I>(&'a self, slot: Slot, stores: I, is_startup: bool) -> usize
+    fn do_shrink_slot_stores<'a, I>(&'a self, slot: Slot, stores: I, _is_startup: bool) -> usize
```

Comment:
remove `_is_startup` arg?

Reply:
To avoid cluttering up this pr, I didn't add that change here. A by product of many locks is that it is harder to hold all of them open the same way we were before at startup. Meanwhile, shrink being called at startup is also tentatively on the chopping block separately, as you and I have been discussing. I was planning to avoid unnecessary code disturbance for the moment. And, maybe I'll add back in a lock held open version of shrink...